### PR TITLE
add sign up link to release notes

### PR DESCRIPTION
**Pull request by jacobprall**
Adds signup link to release notes

### CODE REVIEW

**Review by max-nader — APPROVED** [review]

File: timescaledb/overview/release-notes/index.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ Plans](#future-plans) section. Interested in learning more about what's already
 available? Jump to [What's New](#what-x27-s-new-in-timescaledb-2-5) to see
 what's been released.
 
+<highlight type="note">
+Want to stay up-to-date with our releases? Subscribe to our [release notes email updates](https://www.timescale.com/signup/release-notes).
```
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:9]:
```suggestion
Want to stay up-to-date with new releases? Subscribe to get [release notes email updates](https://www.timescale.com/signup/release-notes).
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: timescaledb/overview/release-notes/index.md (hunk 1 above)
Comment (Rajakavitha1) [reply timescaledb/overview/release-notes/index.md:9]: Suggestion:  Subscribe to [release notes email updates](https://www.timescale.com/signup/release-notes), for the latest releases.

**Review by Rajakavitha1 — APPROVED** [review]

File: timescaledb/overview/release-notes/index.md (hunk 1 above)
Comment (Loquacity) [reply timescaledb/overview/release-notes/index.md:9]: Isn't that a bit repetitive though, since the first sentence says new releases also?

Comment (Loquacity) [reply timescaledb/overview/release-notes/index.md:9]: Can you just make a suggestion using the `Add a suggestion` button? That would make things a lot clearer.

**Pull request merged**